pkg/psmdb/mongos.go

```diff
@@ -134,6 +134,12 @@ func mongosContainer(cr *api.PerconaServerMongoDB) (corev1.Container, error) {
 				ContainerPort: cr.Spec.Sharding.Mongos.Port,
 			},
 		},
+		Env: []corev1.EnvVar{
```

Comment:
can we please please add
```
--- e2e-tests/demand-backup-sharded/run
+++ e2e-tests/demand-backup-sharded/run
@@ -26,6 +26,7 @@ wait_for_running $cluster-rs0 3
 desc 'check if service and statefulset created with expected config'
 compare_kubectl statefulset/$cluster-rs0
 compare_kubectl statefulset/$cluster-cfg
+compare_kubectl deploy/$cluster-mongos

 desc 'write data, read from all'
 run_mongos \
```
line into `e2e-tests/demand-backup-sharded/run`
reason: it is needed to see if such changes modify finial structure in k8s in fact

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/479/commits/e0596be68f275c17289272098117dd3cb4e6f32d